javascript/packages/utils/src/fs.ts

```diff
@@ -72,17 +72,19 @@ export function getCredsFilePath(credsFile: string): string | undefined {
   if (fs.existsSync(credsFile)) return credsFile;
 
   const possiblePaths = [".", "..", `${process.env.HOME}/.kube`];
-  let credsFileExistInPath: string | undefined = possiblePaths.find((path) => {
-    const t = `${path}/${credsFile}`;
-    return fs.existsSync(t);
-  });
+  const credsFileExistInPath: string | undefined = possiblePaths.find(
```

Comment:
Type is not altered on this PR - just the `let` -> to `const` since its not altered in later stage; Again, this PR is strictly only for eslint intro, not refactoring